emote/memory/memory.py

```diff
@@ -169,3 +171,15 @@ def __iter__(self):
 
             data[self.size_key] = self.rollout_count * self.rollout_length
             yield {self.data_group: data, self.size_key: data[self.size_key]}
+
+
+class MemoryWaiter(Callback):
```

Comment:
LGTM but could we come up with a different name? This didn't do what I expected it to do. `MemoryReady` maybe? I'm not sure.

Reply:
`MemoryWarmupWaiter`?

Reply:
Waiter sounds like it will bring me water and get me the check. But sure. I'd be fine with just `MemoryWarmup`.